webapp/src/components/backstage/automation/toggle.tsx

```diff
@@ -37,11 +43,10 @@ const RoundSwitch = styled.span`
     width: 40px;
     height: 24px;
     border-radius: 14px;
-    background: rgba(var(--center-channel-color-rgb), 0.24);
-
+    background: rgba(var(--center-channel-color-rgb), ${(props) => (props.disabled ? '0.08' : '0.24')});
```

Comment:
Super nit: we seem to converge on using destructured arguments in styled-components when it's this short:
```suggestion
    background: rgba(var(--center-channel-color-rgb), ${({disabled}) => (disabled ? '0.08' : '0.24')});
```